src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: $main-color;
+  align-items: center;
+  padding: 0 50px;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__nav-list {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    height: 60px;
+    list-style: none;
+  }
+
+  &__nav-item {
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
```

Comment:
The `&:last-child` selector is used to remove the margin-right, but there is no margin-right set on the `&__nav-item`. Consider adding a margin-right to the `&__nav-item` if you want to use this selector effectively.